### ISSUE TITLE
update tag page wiki excerpting to catch /barnraising as well as /wiki/title type pages

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1039]
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem

This is a one-line change. 

Our tag pages now show the first paragraph of a matching wiki page at the top:

https://publiclab.org/tag/pm

However, some wiki pages are "root" pages without "wiki" in their URL, like:

https://publiclab.org/barnraising

So that code doesn't work, since it's based on the whole URL including "wiki". The code driving this is at: https://github.com/publiclab/plots2/blob/master/app/controllers/tag_controller.rb#L17

```ruby
    @wiki = DrupalNode.where(path: "/wiki/#{params[:id]}").first
```

## Solution

That line should become

```ruby
    @wiki = DrupalNode.where(slug: params[:id]).first
```

This way, it'll work for a page like:

https://publiclab.org/tag/barnraising

and will show the image and correct excerpt from the corresponding wiki page at https://publiclab.org/barnraising.

## Steps to Fix

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/developers) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

**Comment by ViditChopra723** [comment]
Hey can I take a crack at this?

**Comment by jywarren** [comment]
We'd love your help on this and are happy to offer any assistance you might need. Thanks a lot!

Just curious -- we're trying to improve our outreach to new contributors -- how did you happen to hear about us?

**Comment by ViditChopra723** [comment]
Thank you very much! I actually was just looking through the first timers only tag after reading an article about it and decided to try my hand on this since it seemed simple. Here is the article https://medium.com/@kentcdodds/first-timers-only-78281ea47455#.yqtyizf89

If its alright I will read through your links and try to figure out what to do first, if I have any questions I will comment here.

**Comment by ViditChopra723** [comment]
Hey I made a pull request, your link to the devolopers page didnt work however. I think I did it right, any feedback is appreciated.

**Comment by ananyo2012** [comment]
@ViditChopra723 We are reviewing your work. Thanks for your work on this. And thanks for reporting about the developer page. @jywarren The developers page isn't showing up. Need to find the problem.

**Comment by jywarren** [comment]
Super, merged in #1043!
